src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -211,6 +236,9 @@ class GaussianMLPModule(GaussianMLPBaseModule):
                exponential transformation
             - softplus: the std will be computed as log(1+exp(x))
         layer_normalization (bool): Bool for using layer normalization or not.
+        normal_distribution_cls (torch.distribution): normal distribution class
```

Comment:
does this type exist? torch.distribution sounds like a package.

Reply:
torch.distribution doesn't even exist as a package.

please remember that these have to be resolved automatically by the documentation package. it's not sufficient to get it "kind-of" correct or "your know what i mean" correct.

https://pytorch.org/docs/stable/distributions.html#distribution

Reply:
is there a more specific interface you can use? perhaps anything from the Exponential family?

which interface do you actually rely on?